docs/archive/examples/slack-history.md

```diff
@@ -102,9 +102,8 @@ Download \(or copy and paste\) this [html file](https://github.com/airbytehq/air
 
 ## 5. "Productionizing" Saving Slack History
 
-You can find instructions for how to host Airbyte on various cloud platforms [here](../../deploying-airbyte).
+You can find instructions for how to host Airbyte on various cloud platforms [here](../../deploying-airbyte/local-deployment.md).
```

Comment:
this one too feels like it wouldn't make sense to link to a specific deployment option